packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -130,8 +131,23 @@ class Checkbox extends StatefulWidget {
   /// The color to use when this checkbox is checked.
   ///
   /// Defaults to [ThemeData.toggleableActiveColor].
+  ///
+  /// If [fillColor] returns a non-null color in the [MaterialState.selected]
+  /// state, it will be used over this.
   final Color? activeColor;
 
+  /// The color that fills the checkbox when it is checked, in all
+  /// [MaterialState]s.
+  ///
+  /// If this is provided, it will be used over [activeColor].
+  ///
+  /// Resolves in the following states:
+  ///  * [MaterialState.selected].
+  ///  * [MaterialState.hovered].
+  ///  * [MaterialState.focused].
+  ///  * [MaterialState.disabled].
+  final MaterialStateProperty<Color?>? fillColor;
```

Comment:
It would be helpful to provide a class APIDoc example that overrode fillColor. Likewise for Switch. Could be a separate PR.

Reply:
Yes that's a good idea, I will do a followup PR with API docs.